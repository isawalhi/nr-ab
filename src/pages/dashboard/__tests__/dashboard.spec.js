import Dashboard from '../dashboardController'
import DashboardView from '../dashboardView'

describe('Dashboard', () => {
    it('generates html when dashboard is provided correctly', () => {
        expect(new Dashboard().render()).toMatchSnapshot()
    })
})