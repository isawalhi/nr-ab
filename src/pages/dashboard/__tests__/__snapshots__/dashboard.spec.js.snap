// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard generates html when dashboard is provided correctly 1`] = `Promise {}`;
